src/models/encounter.ts

```diff
@@ -0,0 +1,26 @@
+import { Monster } from './monster'
+
+export class Encounter {
```

Comment:
I think we should get adjusted XP in here as mentioned on Discord. From the 2 mins I spent looking at the DMG it should just be totalXP multiplied by some set value based on the monster count